components/CollectiveNavbar.js

```diff
@@ -192,6 +192,7 @@ const DEFAULT_SECTIONS = {
     Sections.CONVERSATIONS,
     Sections.BUDGET,
     Sections.CONTRIBUTORS,
+    Sections.RECURRING_CONTRIBUTIONS,
```

Comment:
Do we really want to show this section for all collectives?

Reply:
I'm not sure. If you let me know which cases we wouldn't want to show it, I can add some logic?

Reply:
Most collectives don't have any recurring contributions, we probably don't want to add it on https://opencollective.com/babel/ for example. An idea would be to show that for collectives that have at least one active recurring contribution. I just checked, and there's only 43 in the database.


```sql
SELECT * FROM "Subscriptions" s
INNER JOIN "Orders" o ON o."SubscriptionId" = s.id
INNER JOIN "Collectives" c ON o."FromCollectiveId" = c.id
WHERE s."isActive" IS TRUE
AND c."type" = 'COLLECTIVE'
```